### ISSUE TITLE
BUG: Serverless Canary Plugin permissions aren't valid for -developer/-admin users

### ISSUE THREAD

**Issue by ryan-roemer** [FormidableLabs/aws-lambda-serverless-reference#39]
Trying this:

```sh
$ STAGE=sandbox aws-vault exec FIRST.NAME-admin --no-session -- \
  yarn lambda:deploy

# ...

  Serverless Error ---------------------------------------
 
  An error occurred: CodeDeployServiceRole - API: iam:CreateRole User: arn:aws:iam::ACCOUNT:user/FIRST.NAME-admin is not authorized to perform: iam:CreateRole on resource: arn:aws:iam::ACCOUNT:role/sls-simple-reference-sandbox-CodeDeployServiceRole-236BX48WS9ZX.
```

Looks like from generated resources from `serverless-plugin-canary-deployments`. What do we need to do to get that hooked up to our `-admin` role to be able to create?

**Comment by tptee** [comment]
The IAM in the canary policy looks right: https://github.com/FormidableLabs/terraform-aws-serverless/blob/master/modules/canary/policy-developer.tf#L31

Wondering if it's not getting attached to the admin role...will check!

**Comment by ryan-roemer** [comment]
Fixed in #36
